### PR TITLE
MM-27352: Add doc for ldap certificate update/delete

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Documents new endpoint for LDAP, TLS Certificate uploading/removing

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-27352

#### Related PR
https://github.com/mattermost/mattermost-webapp/pull/6345
https://github.com/mattermost/mattermost-redux/pull/1224
https://github.com/mattermost/mattermost-server/pull/15361
https://github.com/mattermost/enterprise/pull/752

### CODE REVIEW

File: v4/source/ldap.yaml [hunk 1]
```diff
@@ -191,3 +191,127 @@
...
+      tags:
+        - LDAP
+      summary: Upload public certificate
+      description: >
+        Upload the public certificate to be used for TLS verification. The server will pick a hard-coded filename for the
+        PublicCertificateFile setting in your `config.json`.
+
+        ##### Permissions
+
+        Must have `manage_system` permission.
```
Comment (isacikgoz) [inline v4/source/ldap.yaml:205]: Would it be necessary to add minimum server version here?

**Review by isacikgoz — APPROVED** [review]
LGTM, have a non-blocking question/suggestion below.

File: v4/source/ldap.yaml (hunk 1 above)
Comment (sbishel) [reply v4/source/ldap.yaml:205]: added...

**Review by srkgupta — APPROVED** [review]
LGTM

**Pull request merged**